.github/workflows/ci.yml

```diff
@@ -20,9 +20,32 @@ env:
   CRDS_CLIENT_RETRY_DELAY_SECONDS: 20
 
 jobs:
+  cancel_ci:
+    name: Mandatory checks before CI
+    runs-on: ubuntu-latest
+    outputs:
+      run_next: ${{ steps.skip_ci_step.outputs.run_next }}
+    steps:
+    - name: Set output for skip CI
+      uses: pllim/action-skip-ci@main
```

Comment:
Would be nice if this `action-skip-ci` action was moved to either astropy or spacetelescope, depending on scope.

Reply:
Likely `astropy` but probably won't be resolved for 1-2 weeks until I get okay from Astropy people. This is not the only package using it now, so when that happens, I plan to batch PR the change. But if you are not comfortable, you are welcome to wait as well.

Reply:
Update: `astropy` is using it now, so I think it is just a matter of time that this Action moves into `astropy` org, but it cannot happen before the next infrastructure meeting scheduled for 2021-01-25.

Reply:
Thanks @pllim!  How about we merge this then, and then when it moves you can just add us to your batch PR?

Reply:
Yes, that is my plan. Thanks for the merge. Good to see my random pet project getting some use. 😅

Reply:
Here here to random pet projects!  🎉